neeva/snapshot-release.sh

```diff
@@ -11,15 +11,20 @@ if [ -z "$out" ]; then
 fi
 
 echo "Taking snapshot of $src"
+if [ -e "$out" ]; then 
+  echo -n "$out already exists, hit enter to delete: "
+  read continue_confirmation
+fi
 
+rm -rf $out
 mkdir -p $out
 cp $src/apks/WebLayerSupport.apk $out
 cp $src/apks/WebLayerShell.apk $out
 cp $src/args.gn $out
 
 get_list_of_public_java_sources() {
   # Exclude unneeded browser sandbox code.
-  find org/chromium/weblayer -name \*.java | grep -v BrowserSandboxService.java
+  find org/chromium/weblayer -name \*.java | egrep -v 'BrowserSandboxService.java|BrowserFragmentDelegate.java|BrowserFragmentTabDelegate.java|TabNavigationControllerProxy.java|TabParams.java|TabProxy.java|WebMessageReplyProxyProxy.java'
```

Comment:
nit: you might consider some line wrapping here....

can use "\" if you want

Reply:
@Kokobe - my markdown fu is weak. I meant to say you can use a single backslash "\\" 
```